open_spiel/games/othello.cc

```diff
@@ -173,14 +174,16 @@ CellState PlayerToState(Player player) {
   }
 }
 
-std::string StateToString(CellState state) {
+std::string StateToString(Player player, CellState state) {
```

Comment:
This is kind of odd. Usually you keep the string as absolute but just the observation tensor as player relative.

Reply:
Ok. I reverted this part to keep it absolute.